diff --git a/fairlearn/datasets/_fetch_credit_card.py b/fairlearn/datasets/_fetch_credit_card.py
index 60b6ae1..622f1e3 100644
--- a/fairlearn/datasets/_fetch_credit_card.py
+++ b/fairlearn/datasets/_fetch_credit_card.py
@@ -23,6 +23,8 @@ def fetch_credit_card(*, cache=True, data_home=None, as_frame=True, return_X_y=F
     predictive accuracy of probability of default of credit card clients", Expert Systems
     with Applications, 36(2), 2473-2480, 2009
 
+    .. versionadded:: 0.9.0
+
     Parameters
     ----------
     cache : boolean, default=True
diff --git a/fairlearn/preprocessing/_prototype_representation_learner.py b/fairlearn/preprocessing/_prototype_representation_learner.py
index bb278c9..d9a4748 100644
--- a/fairlearn/preprocessing/_prototype_representation_learner.py
+++ b/fairlearn/preprocessing/_prototype_representation_learner.py
@@ -33,6 +33,8 @@ class PrototypeRepresentationLearner(ClassifierMixin, TransformerMixin, BaseEsti
 
     Read more in the :ref:`User Guide <preprocessing>`.
 
+    .. versionadded:: 0.13.0
+
     Parameters
     ----------
     n_prototypes : int, default=2
